# Default initial positions for cobot_nextup's ros2_control fake system

initial_positions:
  joint1: 0
  joint2: 0
  joint3: 0
  joint4: 0
  joint5: 0
  joint6: 0
  lift_joint: 0